optimization/problems/knapsack/kp_5_80/solutions/solution1/summary.json

```diff
@@ -0,0 +1,29 @@
+{
+  "date": {
+    "value": "2023-10-05"
+  },
+  "objective": {
+    "value": 130,
+    "unit": "",
+    "description": "The objective value of the selected knapsack items."
+  },
+  "approximation ratio": {
+    "value": 1.0,
+    "description": "The depth-one QAOA found the optimal solution and the approximation ratio is 1."
+  },
+  "time to solution": {
+    "value": 2094.261,
+    "unit": "s",
+    "description": "Time-to-solution is reported as the Qiskit Runtime usage and includes the full variational optimization. Qiskit Runtime usage is the time a job spends leveraging the Qiskit Runtime architecture. It includes quantum compute and near-time classical pre- and post-processing. Queue time is not included."
+  },
+    "computational resources": [
+    {
+      "value": 127,
```

Comment:
Same comment as above. It's fine to say that we used an Eagle device (which then implies that the device has 127 qubits). However, I think we should report the actual number of qubits that were used. The description below also states: `"The number of qubits used on the quantum processor to perform the optimization."`